python/ctsm/crop_calendars/cropcal_module.py

```diff
@@ -1,13 +1,17 @@
-# Import the CTSM Python utilities
-import cropcal_utils as utils
-
 import numpy as np
 import xarray as xr
 import warnings
 import sys
 import os
 import glob
 
+# Import the CTSM Python utilities
+_CTSM_PYTHON = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir, os.pardir, "python"
+)
+sys.path.insert(1, _CTSM_PYTHON)
+import ctsm.crop_calendars.cropcal_utils as utils
```

Comment:
Can you speak to why this is needed? In other places we can import stuff throughout the ctsm python library without doing path manipulation - e.g., see job_launcher_qsub.py (for an example in a subdirectory of python/ctsm, like this one is). Is it possible that you need an `__init__.py` in the crop_calendars directory to enable this??

Reply:
Good point; not necessary! I had that bit in my head from adding it to `fsurdatmodifyctsm.py`, but that was only necessary because it's not in `python/`. My latest commit removes these unneeded lines.

Reply:
Cool, thanks.

Reply:
Wait I still see the path logic, am I missing something?

Reply:
This is the thing we should discuss as a group.